sh/health.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+# shellcheck shell=dash
+set -eo pipefail
+
+CHECK="mariadb"
+
+# prefer root if available
+if [ -n "${MYSQL_ROOT_PASSWORD}" ]; then
```

Comment:
I think it's worth providing ENVs for the check, and if they are all empty then it does nothing ?

Reply:
If they are all empty it would try to connect without credentials.